src/pipecat/pipeline/parallel_pipeline.py

```diff
@@ -83,8 +84,8 @@ def __init__(self, *args):
 
         self._up_task = None
         self._down_task = None
-        self._up_queue = asyncio.Queue()
-        self._down_queue = asyncio.Queue()
```

Comment:
If we remove this from here, we will need to also refactor the code below, because we are trying to use both queues here in the `__init__` method.